erpnext/accounts/doctype/pos_session_invoices/pos_session_invoices.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
In a normal use case it would be largely good enough in my opinion. The problem here is that the compliance requirement doesn't consider this good enough at all.

We must ensure that people who want to fraud can't do it, which is much harder :wink: 
And the french anti-fraud department considers that it is the role of the software to ensure, people can't fraud.

As of now, when one uses the POS, he can easily delete the invoice after the transaction by clearing LocalStorage. We should have a cryptographic log ensuring that if they do it, at least we can prove it.

A system manager can also get the authorization to delete items in the "Deleted Document" which is not considered acceptable by the government.

Numbering series are a good way to identify missing invoices and are also a legal requirement, the only issue here is that people can rename their invoices if they want to fraud. So we need to ensure that if they do so, we have a trace...

I would prefer people didn't fraud in the first place :smile: